velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -456,5 +456,52 @@ TEST_F(DateTimeFunctionsTest, quarterDate) {
   EXPECT_EQ(3, quarter("1954-08-08"));
 }
 
+TEST_F(DateTimeFunctionsTest, monthsBetween) {
+  const auto monthsBetweenFunc = [&](const std::optional<Timestamp> timestamp1,
+                                     const std::optional<Timestamp> timestamp2,
+                                     const std::optional<bool> roundOff) {
+    return evaluateOnce<double>(
+        "months_between(c0, c1, c2)", timestamp1, timestamp2, roundOff);
+  };
+
+  std::string zone = "America/Los_Angeles";
+
+  setQueryTimeZone(zone);
+
+  auto monthsBetween = [&](const auto& inputDate1,
+                           const auto& inputDate2,
+                           const auto& roundOff) {
+    Timestamp timestamp1 = util::fromTimestampString(inputDate1);
+    timestamp1.toGMT(util::getTimeZoneID(zone));
```

Comment:
perhaps, extract a variable to store util::getTimeZoneID(zone)

Reply:
@mbasmanova done